lib/util.ts

```diff
@@ -82,6 +82,14 @@ export function toAbsoluteUrl(url = '', baseUrl) {
   return url[0] === '/' ? `${baseUrl}${url}` : `${baseUrl}/${url}`;
 }
 
+export function toRelativeUrl(url = '', baseUrl) {
```

Comment:
where is this function being used?

Reply:
It's mostly for the downstream SDKs, as they both depend on auth-js, looks like here can be a good place for some common util functions.